apps/backoffice-v2/src/pages/Entity/hooks/useEntity/kycBlock.stories.tsx

```diff
@@ -0,0 +1,279 @@
+                        format: 'text',
+                        pattern: '',
+                        isEditable: false,
+                        dropdownOptions: undefined,
+                      },
+                      {
+                        title: 'Result',
+                        value: 'approved',
+                        type: 'text',
+                        format: 'text',
+                        pattern: '',
+                        isEditable: false,
+                        dropdownOptions: undefined,
+                      },
+                      {
+                        title: 'Selfie face match',
+                        value: 'approved',
+                        type: 'text',
+                        format: 'text',
+                        pattern: '',
+                        isEditable: false,
+                        dropdownOptions: undefined,
+                      },
+                      {
+                        title: 'Issues',
+                        value: 'none',
+                        type: 'text',
+                        format: 'text',
+                        pattern: '',
+                        isEditable: false,
+                        dropdownOptions: undefined,
+                      },
+                      {
+                        title: 'Full report',
+                        value: 'https://www.veriff.com',
+                        valueAlias: 'View',
+                        type: 'text',
+                        format: 'text',
+                        pattern: '',
+                        isEditable: false,
+                        dropdownOptions: undefined,
+                      },
+                    ],
+                  },
+                },
+              ],
+            },
+          ],
+        },
+        {
+          type: 'multiDocuments',
+          value: {
+            isLoading: false,
+            data: [
+              {
+                title: 'passport',
+                imageUrl: 'https://picsum.photos/400/600',
+                fileType: 'image/png',
+              },
+            ],
+          },
+        },
+      ],
+    },
+  ]);
+
+export const KycBlock = () => {
+  const kycBlocks = useKycBlocks(childWorkflows);
+
+  return (
+    <QueryClientProvider client={queryClient}>
+      <div className={`flex h-screen flex-col space-y-8 overflow-y-auto p-4`}>
+        {kycBlocks?.map((kycBlock, index) => (
+          <Card className={`me-4`} key={`task:${index}`}>
+            <CardContent className={ctw('grid gap-2')}>
+              {kycBlock?.map((cell, index) => {
+                const Cell = cells[cell?.type];
+
+                return (
+                  <Cell key={`task-cell:${cell?.type}:${cell?.id ?? ''}:${index}`} {...cell} />
+                );
+              })}
+            </CardContent>
+          </Card>
+        ))}
+      </div>
+    </QueryClientProvider>
+  );
+};
+
+export const withTanStackQuery = Story => {
```

Comment:
We have cells in use that use `@tanstack/react-query` and `react-router-dom`, otherwise they break Storybook.